src/openrct2/ride/coaster/JuniorRollerCoaster.cpp

```diff
@@ -1844,12 +1844,12 @@ static constexpr const uint32_t junior_rc_track_pieces_diag_blockbrakes[2][4] =
     },
 };
 
-template<JuniorRCSubType subType> constexpr uint8_t JuniorRCGetSubTypeOffset(const TrackElement& trackElement)
+template<TJuniorRCSubType subType> constexpr uint8_t JuniorRCGetSubTypeOffset(const TrackElement& trackElement)
```

Comment:
I think you misunderstood me, sorry if I didn't make it more clear, it should be `template<JuniorRCSubType TSubType>`, that is how we have it in some other places.